lbrynet/extras/daemon/Daemon.py

```diff
@@ -3258,179 +3258,108 @@ def jsonrpc_routing_table_get(self):
-                comments = comments[page_size * (page - 1): page_size * page]
-        # By now comments should be a list containing comment dicts that are supposed to be
-        # at the given height that was requested. The parent_id may or may not be present
-        # in the dicts, as they may or may not be replies to comments at a higher level
-        # However this is dependent purely on whether or not parent_comment_id is None or not
-        reply_lists = await jsonrpc_batch(url, [
-            rpc_body('get_comment_replies', index, comm_index=comment['comment_id'])
-            for index, comment in enumerate(comments)
-        ])
-        response = {
-            'page': page,
-            'page_size': -1 if page_size is None else page_size,
-            'comments': []
-        }
-        if flat:
-            # If it's flat then we'll need to get the comments into an order such that
-            # If an element e in the list has a non-null parent id, the element before it
-            # is either also a reply with the same parent id, or has an id that equals e's parent id,
-            # in which case it's the comment that is being replied to.
-            # Otherwise, if it has a null parent id, then it is a top level comment.
-
-            # To do this, we create a dict that maps the index of the comment in the array
-            # to a list containing the comment IDs of the replies
-            comment_replies = {resp['id']: resp['result'] for resp in reply_lists if 'result' in resp}
-
-            # Next, we create a batch request for the actual data of all of the replies
-            # the id in this batch request is going to be in the form 'X:Y'
-            # where X is the index of the parent comment in `comments,
-            # and Y is index of the reply's ID within the list X maps to in `comment_replies`
-            full_replies_batch = [
-                rpc_body('get_comment_data', f'{parent_idx}:{idx}', comm_index=reply_id, better_keys=True)
-                for parent_idx, id_list in comment_replies.items()
-                for idx, reply_id in enumerate(id_list[0:max_replies_shown])
+            (list)  Containing comments stored as dictionary objects:
+            [
+                {
+                    "comment":      (str) The actual string as inputted by the user,
+                    "comment_id":   (str) The Comment's unique identifier,
+                    "channel_name": (str) Name of the channel this was posted under, prepended with a '@',
+                    "channel_id":   (str) The Channel Claim ID that this comeent was posted under,
+                    "signature":    (str) The signature of the comment,
+                    "channel_uri":  (str) Channel's URI in the ClaimTrie,
+                    "parent_id":    (str) Comment this is replying to, (None) if this is the root,
+                    "timestamp":    (int) The time at which comment was entered into the server at, in nanoseconds.
```

Comment:
should this be a linux epoch? I'm seeing 1559075031817770554 which doesn't convert (https://www.epochconverter.com/) like the release_time attribute we have. Is it becuase it's in nano seconds? ms are probably fine/better to be consistent with other timestamps.

Reply:
the server uses nanoseconds yea

Reply:
should be seconds to be consistent with everything else

Reply:
Okay so I have:
- nanoseconds: 1
- milliseconds: 1
- seconds: 1

Can we please come to a consensus?

Reply:
Never mind i just made it seconds, should be good